### ISSUE TITLE
we cannot bookmark Java connection URLs and open them in another browser

### ISSUE THREAD

**Issue by jstrachan** [hawtio/hawtio-kubernetes#70]
a URL like http://fabric8.vagrant.f8/?con=jr-test1-xeuag links to a pod without the namespace; so its a bit hard if there's no jolokia URL in localstorage to figuring it out. 

Maybe if the URL also had the namespace inside it; the code could check if there's an entry in localStorage for the jolokia URL and if not just create it on the fly?

**Comment by jimmidyson** [comment]
Why do we need to store in localstorage at all? I'd suggest having the namespace & name in the URL - otherwise noone can send links round to others to browse.

**Comment by jstrachan** [comment]
agreed - I think the URL parameters should be enough really. (The localStorage is historical I think from when we had to do a network request to find the URL etc)

**Comment by davsclaus** [comment]
yeah would be nice with bookmarks. Also maybe even a bookmark for a RC, that takes you to a page that shows the pods (if more than 2+) and allows to select which pod to connect to. As the pods can come and go, then the url can become stale over a time. But the url for the RC can be stale.
